README.md

```diff
@@ -250,31 +247,6 @@ Simply run:
 generic-worker --config CONFIG-FILE
 ```
 
-and watch logs for a successful startup. If you can see it is polling the Queue, and the process does not exit, then you can continue. If it reports a problem, follow any instructions it provides. If you are really stuck, join #taskcluster channel on irc.mozilla.org, and ask for help.
```

Comment:
I only removed this because it seemed much of the logging was specific about Azure, which no longer should happen.  I've been trying to get a config put together to run the worker locally to see what's logged now to put something back in here, but haven't had luck so far.